slapp/utils/query_utils.py

```diff
@@ -17,22 +43,27 @@ def get_labeling_db_credentials() -> dict:
 
     Raises
     ------
-    LabelingEnvException
-        Raised if LABELING_USER/LABELING_PASSWORD are not set.
+    CredentialsException
+        Raised if user and password are not set as ENV variables
```

Comment:
Typically we use all caps for env variables. I think "user" and "password" are a little generic, so I'd prefer the prior version (LABELING_USER, LABELING_PASSWORD).

Reply:
That prior behavior is still in place and that structure of the variable names is still there. I just generalized this function with `env_prefix` so we can target different postgres servers. For example, now I have `LABELING_USER` and `LIMS_USER`. You can see these [here](https://github.com/AllenInstitute/segmentation-labeling-app/blob/bcdead70c5629651c1c1b212d47e1c855c0b86d2/slapp/data_selection/select_data.py#L88-L95)

Reply:
oh, I see, I misinterpreted. I find the documentation a little confusing though. The part about the list of keys is not in the section where you describe the format of the env variables. The code does uppercase the variables, but they're actually supposed to be all-caps when set in the env variable, so I think it's less confusing to document them as capitals (especially since they're not user-set in the function).

I would update it to be something like this:

```
    """Get labeling DB credentials from environment variables.
Environment variables must be in the format `{env_prefix}_{key}`,
where `key` is one of the following (case-sensitive): ['USER', 'PASSWORD', 'HOST', 'PORT', 'DATABASE']. Examples: LABELING_USER, LABELING_PASSWORD.
If values for `host`, `port`, and `database` are not passed as function arguments, then they must also be
set as environment variables using the `{env_prefix}_{key}` format, e.g. LABELING_HOST.

    Parameters
    ----------
    env_prefix : str
        expected environment variable prefix for credential keys.
        expected ENV variables are <prefix><KEY>. Values for <KEY> include the following: ["HOST", "PORT", "DATABASE", "USER", "PASSWORD"]. 
```

Reply:
Updated the docstring.